client/lib/domains/email-forwarding/reducer.js

```diff
@@ -118,6 +118,8 @@ function reducer( state, payload ) {
 		case UpgradesActionTypes.EMAIL_FORWARDING_DELETE_COMPLETED:
 			state = deleteTemporaryMailbox( state, action.domainName, action.mailbox );
 			break;
+		case UpgradesActionTypes.EMAIL_FORWARDING_RESEND_VERIFICATION_COMPLETED:
```

Comment:
Shouldn't you, like, _do_ something here? :grin: Otherwise I don't quite see the reason for this event (but maybe I'm missing something).